src/components/IndividualMovieDetails.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import '../styles/IndividualMovie.scss'
+import '../styles/Trailer.scss'
+
+const IndividualMovieDetails = props => {
```

Comment:
Nice add to hold all our movie details to clean up the code for developer empathy